src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -34,6 +45,20 @@ abstract class AbstractFunctionalTest extends Specification {
 
     String dockerServerUrl
 
+    @Shared
+    DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder()
+        .withDockerHost("unix:///var/run/docker.sock")
```

Comment:
Now tests won't run on Windows. It would be better to use `TestConfiguration` here.